src/shared/utils/helpers/format-past-date.ts

```diff
@@ -2,8 +2,23 @@ import formatDistanceStrict from "date-fns/formatDistanceStrict";
 import parseISO from "date-fns/parseISO";
 
 export default function (dateString?: string) {
-  const parsed = parseISO((dateString ?? Date.now().toString()) + "Z");
-  return formatDistanceStrict(parsed, new Date(), {
-    addSuffix: true,
-  });
+  if (!dateString || dateString === undefined) {
```

Comment:
The `|| dateString === undefined` part shouldn't be necessary. `undefined` is falsey in JS, thus `!dateString` will evaluate to true if it is undefined.
```suggestion
  if (!dateString) {
```